index.js

```diff
@@ -45,7 +49,10 @@ exports.parse = function (str) {
 	return ret;
 };
 
-exports.stringify = function (obj) {
+exports.stringify = function (obj, opts) {
+	opts = opts || {};
+	opts.strict = opts.strict !== false;
```

Comment:
Don't overwrite the user supplied options object. Objects are reference based, so this will change the behavior if the user calls multiple time with the same object. Instead just assign it to a variable.